### DISCUSSION TITLE
What does "experimental" mean?

### DISCUSSION THREAD

**Discussion by serranoarevalo** [prisma/prisma#2080] [Q&A] [answered]
I see that Prisma Migrate is on "experimental" , what does that mean? alpha? beta?
What's timeframe for Prisma Migrate?
Thanks!

**Comment by nikolasburk** [comment, marked as answer]
Hey Nicolás, nice to see you around in our Prisma 2.0 community! 🙌 

Experimental basically means that it's very early and _might_ still significantly change. We currently don't map it to alpha or beta terminology but I guess you can imagine it as an alpha version that should not be used in production yet! 🙂

**Reply by serranoarevalo** [reply]
Thanks Nikolas! Do you have any timeline for Prisma Migrate?

**Reply by nikolasburk** [reply]
It's currently _planned_ for Q2, but it may take longer. Unfortunately, we can't commit to a concrete timeline right now...

**Comment by RyanEdwardHall** [comment]
Given the time that's elapsed since the question was originally asked, I'm curious if the timeline around removing the experimental flag has solidified? Thanks!

**Reply by nikolasburk** [reply]
Hey @RyanEdwardHall, we indeed have some more information about the timeline on our [roadmap](https://www.notion.so/prismaio/Prisma-public-roadmap-50766227b779464ab98899accb98295f).

Two items related to Prisma Migrate might be interesting:

- [Simplify database migrations during prototyping](https://www.notion.so/prismaio/Simplify-database-migrations-during-prototyping-2f24abdd283c46f79c688a7cd99fc0b6) (Status: **WIP**)
- [First Beta version of Prisma Migrate](https://www.notion.so/prismaio/Migrate-Beta-f44c83316a9f43f98ebd9cf562f302a9) (Status: **Planned**)

While we can't commit to concrete dates, the [terminology](https://www.notion.so/prismaio/More-details-754fa5db113e45f7aca1954c57b6247c#e358fe2638f546fa804cece2008c3ce5) means that _WIP_ items are **typically** released within the next ~2 months and _planned_ items are generally expected within a period of 3 to 6 months. 

Hope that helps 🙌

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
